components/results_table/commons/results_table.lua

```diff
@@ -0,0 +1,141 @@
+	end
+
+	header:tag('th'):css('width', '420px'):attr('colspan', 2):wikitext('Tournament')
+
+	if self.config.queryType ~= Opponent.team then
+		header:tag('th'):css('min-width', '70px'):wikitext('Team')
+	elseif self.config.playerResultsOfTeam then
+		header:tag('th'):css('min-width', '105px'):wikitext('Player')
+	end
+
+	if not self.config.hideResult then
+		header:tag('th'):css('min-width', '105px'):attr('colspan', 2):wikitext('Result')
+	end
+
+	header:tag('th'):attr('data-sort-type', 'currency'):wikitext('Prize')
+
+	return header
+end
+
+function ResultsTable:buildRow(placement)
+	local placementCell = mw.html.create('td')
+	Placement._placement{parent = placementCell, placement = placement.placement}
+
+	local row = mw.html.create('tr')
+		:addClass(self:rowHighlight(placement))
+		:tag('td'):wikitext(placement.date:sub(1, 10)):done()
+		:node(placementCell)
+
+	local tierDisplay, tierSortValue = self:tierDisplay(placement)
+
+	row:tag('td'):attr('data-sort-value', tierSortValue):wikitext(tierDisplay)
+
+	if self.config.gameIconsData then
+		row:tag('th'):node(self:gameIcon(placement))
+	end
+
+	local tournamentDisplayName = BaseResultsTable.tournamentDisplayName(placement)
+
+	row
+		:tag('td'):css('width', '30px'):attr('data-sort-value', tournamentDisplayName):wikitext(LeagueIcon.display{
+			icon = placement.icon,
+			iconDark = placement.icondark,
+			link = placement.parent,
+			name = tournamentDisplayName,
+			options = {noTemplate = true},
+		}):done()
+		:tag('td'):attr('data-sort-value', tournamentDisplayName):css('text-align', 'left'):wikitext(Page.makeInternalLink(
+			{},
+			tournamentDisplayName,
+			placement.pagename
+		))
+
+	if self.config.playerResultsOfTeam or self.config.queryType ~= Opponent.team then
+		row:tag('td'):css('text-align', 'right'):attr('data-sort-value', placement.opponentname):node(self:opponentDisplay(
+			placement,
+			{flip = true, teamForSolo = not self.config.playerResultsOfTeam}
+		))
+	end
+
+	if not self.config.hideResult then
+		local score, vsDisplay = self:_processVsData(placement)
+		row
+			:tag('td'):wikitext(score):done()
+			:tag('td'):css('text-align', 'left'):node(vsDisplay)
+	end
+
+	row:tag('td'):css('text-align', 'right'):wikitext('$' .. Currency.formatMoney(
+			self.config.queryType ~= Opponent.team and placement.individualprizemoney
+			or placement.prizemoney
+		))
+
+	return row
+end
+
+function ResultsTable:_processVsData(placement)
+	local lastVs
+	if Table.isEmpty(placement.lastvsdata) then
+		lastVs = self:processLegacyVsData(placement)
+	else
+		lastVs = placement.lastvsdata
+	end
+
+	if String.isNotEmpty(lastVs.groupscore) then
+		return placement.groupscore, Abbreviation.make('Grp S.', 'Group Stage')
+	end
+
+	local score = (placement.lastscore or '-') .. SCORE_CONCAT .. (lastVs.score or '-')
+	local vsDisplay = self:opponentDisplay(lastVs, {})
+
+	return score, vsDisplay
+end
+
+-- overwritable
+function ResultsTable:processLegacyVsData(placement)
```

Comment:
You could have this in the base instead, have it automatically created (if missing). So that the display Modules just don't need to worry about this edge

Reply:
the idea behind putting it here was that it is not needed in the awards version
but we can move it if wanted

Reply:
Fits better there. Also if there's a third type in the future which has a use for it, wouldn't need to repeat the logic